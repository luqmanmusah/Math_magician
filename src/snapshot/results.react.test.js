import React from 'react';
import renderer from 'react-test-renderer';
import Result from '../components/result';

it('renders correctly', () => {
  const tree = renderer
    .create(<Result result="5" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
