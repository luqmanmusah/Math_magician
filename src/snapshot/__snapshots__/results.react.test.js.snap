// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="result"
>
  <p>
    5
  </p>
</div>
`;
